cmd/queue/main.go

```diff
@@ -26,6 +26,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
Do you mind collapsing the imports here? (we should have a bot for this... @mattmoor another socketpuppet?)

Reply:
done!